docs/cargo-llvm-cov-show-env.txt

```diff
@@ -8,6 +8,10 @@ OPTIONS:
         --export-prefix
             Prepend "export " to each line, so that the output is suitable to be sourced by bash
 
+        --export-pwsh-prefix
```

Comment:
"--export-prefix" is a name from it emits env vars with "export " prefix. ("export" here is a noun, not a verb)

So, naming something like "--pwsh-env-prefix" (or "--with-pwsh-env-prefix" discussed in https://github.com/taiki-e/cargo-llvm-cov/issues/320#issuecomment-1816773684) is better here.